### ISSUE TITLE
Crash When getting started with Mapbox

### ISSUE THREAD

**Issue by tarakinsuide** [mapbox/mapbox-gl-native#12653]
Hi,

I'm following the same steps as mentioned in getting started with android post on your site https://www.mapbox.com/help/first-steps-android-sdk/

But my app crashes and here is the log:
```

    --------- beginning of crash
08-15 22:37:42.126 20794-20794/com.insuide.tarak.gitex20181 E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.insuide.tarak.gitex20181, PID: 20794
    java.lang.RuntimeException: Unable to create service com.mapbox.android.telemetry.TelemetryService: java.lang.SecurityException: "passive" location provider requires ACCESS_FINE_LOCATION permission.
        at android.app.ActivityThread.handleCreateService(ActivityThread.java:3363)
        at android.app.ActivityThread.-wrap4(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1689)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:172)
        at android.app.ActivityThread.main(ActivityThread.java:6590)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.SecurityException: "passive" location provider requires ACCESS_FINE_LOCATION permission.
        at android.os.Parcel.readException(Parcel.java:2005)
        at android.os.Parcel.readException(Parcel.java:1951)
        at android.location.ILocationManager$Stub$Proxy.requestLocationUpdates(ILocationManager.java:691)
        at android.location.LocationManager.requestLocationUpdates(LocationManager.java:900)
        at android.location.LocationManager.requestLocationUpdates(LocationManager.java:471)
        at com.mapbox.android.core.location.AndroidLocationEngine.requestLocationUpdates(AndroidLocationEngine.java:103)
        at com.mapbox.android.telemetry.TelemetryService.onConnected(TelemetryService.java:94)
        at com.mapbox.android.core.location.AndroidLocationEngine.activate(AndroidLocationEngine.java:75)
        at com.mapbox.android.telemetry.TelemetryService.activateLocationEngine(TelemetryService.java:186)
        at com.mapbox.android.telemetry.TelemetryService.connectLocationEngine(TelemetryService.java:173)
        at com.mapbox.android.telemetry.TelemetryService.registerLocationReceiver(TelemetryService.java:164)
        at com.mapbox.android.telemetry.TelemetryService.createLocationReceiver(TelemetryService.java:160)
        at com.mapbox.android.telemetry.TelemetryService.onCreate(TelemetryService.java:41)
        at android.app.ActivityThread.handleCreateService(ActivityThread.java:3353)
        at android.app.ActivityThread.-wrap4(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1689) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:172) 
        at android.app.ActivityThread.main(ActivityThread.java:6590) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
08-15 22:37:44.029 20794-20799/com.insuide.tarak.gitex20181 I/zygote64: Do partial code cache collection, code=29KB, data=19KB
    After code cache collection, code=29KB, data=19KB
    Increasing code cache capacity to 128KB

```

**Comment by tobrun** [comment]
cc @electrostat

**Comment by zugaldia** [comment]
Thank you @tarakinsuide - could you share more information about which version of our Android SDK and which version of Android are you using to run the example? Thanks!

**Comment by tobrun** [comment]
I'm unable to reproduce this error on a clean project using sdk 6.4.0 (and not including any location permissions). Not including the Mapbox.getInstance invocation results in a different exception. Any additional information would be great @tarakinsuide!

**Comment by vikram2330** [comment]
This happens if you remove the location permissions manually from setting and come back to application. 
Resolved it by adding 
TelemetryEnabler.updateTelemetryState(TelemetryEnabler.State.DISABLED)
after initialising Mapbox sdk

**Comment by zugaldia** [comment]
@electrostat once you confirm the issue, could you cut a ticket on https://github.com/mapbox/mapbox-events-android?

**Comment by electrostat** [comment]
@vikram2330 we released this fix in the latest Telemetry build `3.1.5` : https://github.com/mapbox/mapbox-events-android/pull/184 . This was included in the latest MapSDK `6.4.0`.

We had another client report this issue, which was replicable when disabling location permissions manually and then re-entering a non-closed app with our sdk. I added in a check that will prevent this.

**Comment by tarakinsuide** [comment]
Hi, 
Got my issue solved. I disabled the location permission by mistake!
